tracer/src/Datadog.Trace.ClrProfiler.Managed.Loader/Startup.cs

```diff
@@ -67,7 +72,7 @@ static Startup()
             }
         }
 
-        internal static string ManagedProfilerDirectory { get; }
+        internal static string? ManagedProfilerDirectory { get; }
```

Comment:
If you want, you can avoid this `?` if you assign the result of `ResolveManagedProfilerDirectory()` to a local variable instead.